pkgs/games/voxelands/default.nix

```diff
@@ -17,12 +34,24 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    cmake irrlicht libpng bzip2 libjpeg sqlite
-    libXxf86vm libGLU libGL openal libvorbis xlibsWrapper pkgconfig
+    bzip2
+    cmake
```

Comment:
cmake and pkg-config should be in `nativeBuildInputs`

Reply:
Addressed in 45f32f9e82b79d49c1b11e1c4630463cbde14168